apis/kueue/v1beta1/constants.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+const (
+	ResourceInUseFinalizerName = "kueue.k8s.io/resource-in-use"
```

Comment:
Is this domain correct?
Probably, `kueue.x-k8s.io/resource-in-use`?

Reply:
Ah, this is my misunderstanding.
Since the API will be graduated to beta, `kueue.k8s.io/resource-in-use` is correct.

Reply:
We'll stay in x-k8s. We have consulted with @liggitt in any case, to make sure we are following best practices in our first beta release.

Reply:
Sounds good. Thanks for letting me know.

Reply:
I swear I had changed it before :|

It should be `kueue.x-k8s.io`. Fixed.